tools/crm_resource_runtime.c

```diff
@@ -1241,7 +1242,7 @@ cli_resource_restart(resource_t * rsc, const char *host, int timeout_ms, cib_t *
             goto failure;
         }
 
-    } while(g_list_length(list_delta) > 0);
```

Comment:
You can find stuff like
```
while (isspace(*buffer++)) /* omitted */ ;
```
every here and there.
It's entirely OK, although ambiguous to the human comprehension, especially if formatted in such a suggestive manner.